docs/guides/privacy.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Run a privacy report
+---
+
+# Run a privacy report
+
+Bearer CLI's [privacy report type](/explanations/reports/#privacy-report) allows anyone on your team to generate information about the data types processed by your app, as well as any third-party libraries or APIs that receive data. This enables you to do things like:
```

Comment:
Bearer CLI's [privacy report type](/explanations/reports/#privacy-report) allows you to collect information (sensitive data, data subjects, internal and third-party components processing sensitive data) that are required to comply with numerous privacy laws. This enables you to do things like:
- Create a record of processing activities (ROPA) for GDPR.
- Create snapshots of data processing over time.
- Compare sensitive data processing across multiple applications.
- Identify existing and new third parties that process sensitive data.
- Detect oversharing of sensitive data.

Reply:
updated